Sources/PubNub/EventEngine/Core/Dispatcher.swift

```diff
@@ -83,8 +81,14 @@ class EffectDispatcher<Invocation: AnyEffectInvocation, Event, Dependencies>: Di
       effect: effect,
       with: id
     )
-    PubNub.log.debug(
-      "Dispatching effect \(effect)",
+    logger.debug(
```

Comment:
This is internal logic, should we use `trace` level here? Because for debug it is too verbose.